charts/newrelic-k8s-metrics-adapter/templates/deployment.yaml

```diff
@@ -64,6 +64,10 @@ spec:
             secretKeyRef:
               name: {{ template "common.naming.fullname" . }}
               key: personalAPIKey
+        {{- with (include "common.proxy" .) }}
```

Comment:
Anything to be added in the values.yaml and the chart readme (that should be auto-generated) about this change? 
Possibly depending on https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/43

Reply:
Good point. As I am finishing the e2e test, let's wait If https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/43 is merged by then and I can add it on top of it. WDYT?

Reply:
👍